themes/default/layouts/index.html

```diff
@@ -271,29 +267,47 @@ <h6>{{ $item.title }}</h6>
         </div>
     </section>
 
-    <!-- Benefits -->
-    <section class="container mx-auto my-12 lg:my-24 relative">
-        <h2 class="text-center">{{ .Params.benefits.title }}</h2>
-        <div class="benefits-section relative">
-            <div class="shape-background home-benefits-section">
-                <div class="shape-container">
-                    <div class="circle-left"></div>
-                    <div class="circle-right"></div>
+    <section class="container mx-auto my-16 text-center px-3">
+        <div class="card bg-violet-200 p-6 rounded-2xl">
+            <h2 class="rainbow-text">Doing more with less</h2>
```

Comment:
All other section header verbs are in the imperative -- we should follow that here also:

```suggestion
            <h2 class="rainbow-text">Do more with less</h2>
```